frontend/src/pages/projects/utils.ts

```diff
@@ -9,6 +9,7 @@ import { NotebookState } from './notebook/types';
 
 export const getDisplayNameFromK8sResource = (resource: K8sDSGResource): string =>
   resource.metadata.annotations?.['openshift.io/display-name'] || resource.metadata.name;
+
```

Comment:
Is this newline intentional?

Reply:
I'll remove. I originally had changes in this file that got backed out in subsequent revisions.